pkg/resources/api_integration.go

```diff
@@ -212,10 +213,6 @@ func ReadAPIIntegration(d *schema.ResourceData, meta interface{}) error {
 					return err
 				}
 			}
-		case "API_KEY":
```

Comment:
why is this being removed?

Reply:
@sfc-gh-swinkler  API_KEY is a sensitive value in Snowflake, so we can't Read it (back) with Terraform. 
* See also the user resource which also omits the password property from the Read. 
* Integration Test error: https://github.com/Snowflake-Labs/terraform-provider-snowflake/actions/runs/3641738297/jobs/6148030735#step:9:284

Reply:
thank you for the clarification. approved